addons/token-exchange/agent_mirrorpeer_controller.go

```diff
@@ -102,6 +126,13 @@ func (r *MirrorPeerReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 		}
 	}
 
+	klog.Infof("labeling rbd storageclasses")
+	errs = r.labelRBDStorageClasses(ctx, mirrorPeer, scr.Namespace)
```

Comment:
I would label CephFS storage class as well as MetroDR would use the same scheme for CephFS (which is supported)

Reply:
added to label cephfs storageclasses too